### DISCUSSION TITLE
Changing internal database to ETCd

### DISCUSSION THREAD

**Discussion by evrynet1** [gravitational/teleport#37578] [Q&A] [answered]
Hello,

How can we move the current internal database in a standalone setup over to a freshly setup ETCd HA solution?

Thank you

**Comment by stevenGravy** [comment, marked as answer]
You can extract the most important parts like the certificates and some other settings with `tctl get all`. With a etcd (would also recommend looking at the postgres option) you can use the bootstrap option to start with those values. Not all values are extracted with `tctl get all` so expect it will take some iterations to migrate everything. 

Please see https://goteleport.com/docs/management/operations/backup-restore/ for more info on extracting and migrating.
